src/lib/rule-finder.js

```diff
@@ -51,8 +59,13 @@ function _getPluginRules(config) {
 }
 
 function _getAllAvailableRules(pluginRules) {
-  var allRules = fs
-    .readdirSync('./node_modules/eslint/lib/rules')
+  var allRules
+  var eslintPath = require.resolve('eslint')
+  eslintPath = eslintPath.match(/.*eslint/)[0]
+  // no check required for existence of eslint here
+  // refer line#9-line#14
+  allRules = fs
+    .readdirSync(eslintPath)
```

Comment:
Does this really resolve to the *lib/rules* subdir? Rather looks like just eslint's base path.